modules/openxr/register_types.cpp

```diff
@@ -54,49 +54,55 @@ static void _editor_init() {
 
 #endif
 
-OpenXRAPI *openxr_api = nullptr;
-Ref<OpenXRInterface> openxr_interface;
+static OpenXRAPI *openxr_api = nullptr;
+static Ref<OpenXRInterface> openxr_interface;
 
-void preregister_openxr_types() {
-	// For now we create our openxr device here. If we merge it with openxr_interface we'll create that here soon.
+void initialize_openxr_module(ModuleInitializationLevel p_level) {
+	if (p_level == MODULE_INITIALIZATION_LEVEL_SERVERS) {
```

Comment:
I need to check but I think we've moved this too far down, this part needs to run really early and I think was run during the pre-register functions. Now it's run later if I read the code change in `main.cpp` correctly.

That said, the only option earlier is `MODULE_INITIALIZATION_LEVEL_CORE` but this is run before core extensions are created. Might still be correct for OpenXR but not sure. 

I'll do some testing tomorrow, it's bed time for me.